src/BinSkim.Rules/PERules/BA2006.BuildWithSecureTools.cs

```diff
@@ -106,10 +107,9 @@ public override void AnalyzePortableExecutableAndPdb(BinaryAnalyzerContext conte
 
             var goodCompilers = new HashSet<string>();
 
-            var badModules = new List<ObjectModuleDetails>();
             StringToVersionMap allowedLibraries = context.Policy.GetProperty(AllowedLibraries);
 
-            var languageToBadModules = new Dictionary<Language, List<ObjectModuleDetails>>();
+            var languagesToBadModules = new Dictionary<Language, List<ObjectModuleDetails>>();
```

Comment:
>languagesToBadModules [](http://example.com/codeflow?start=16&length=21)

please revert this name, the pattern is key type + value type.

Reply:
done